scripts/modules_auto_pr/modules_auto_pr.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e  # Exit immediately if a command exits with a non-zero status
+set -x  # Print commands and their arguments as they are executed
+
+####################################################################################################
+#                                            VARIABLES                                             #
+####################################################################################################
+
+DATE=$(date +"%Y-%m-%d_%Hh%M")
+REPO_URL="git@github.com:hpcugent/vsc_user_docs.git"
+BASE_BRANCH="main"
+BRANCH_NAME="auto_update_modules_$DATE"
+REPO_NAME="vsc_user_docs" # script available_modules.py requires this to be the name. Do not change.
+REPO_PATH="$VSC_SCRATCH/modules_auto_pr_script_$DATE/$REPO_NAME"
```

Comment:
Use `/tmp/$USER`?

Reply:
Fixed in [use /tmp/user as temp dir](https://github.com/hpcugent/vsc_user_docs/pull/680/commits/4bd6e8a386356da0c73c5d637e8c8f848e6d827e)